prow/cluster/private/authentikos_deployment.yaml

```diff
@@ -41,6 +41,16 @@ spec:
         args:
         - --verbose
         - --secret=authentikos-token
+        - --creds=/etc/creds/service-account.json
         - --namespace=test-pods
         - --scopes=https://www.googleapis.com/auth/devstorage.full_control
         - "--format=Authorization: Bearer %v"
+        volumeMounts:
+        - name: creds
+          mountPath: /etc/creds
+          readOnly: true
+      volumes:
+      - name: creds
+        secret:
+          defaultMode: 0644
+          secretName: service-account
```

Comment:
We should probably be using a separate service account and/or token instead of using the `service-account` secret for everything. If this is supposed to be temporary I guess it doesn't matter much.
/shrug

Reply:
Yes, it is temporary otherwise I would change it. Also this runs in the private cluster where there is one SA tuned specifically for storage, so not crucial for separation concerns.